commands/keys/index.js

```diff
@@ -0,0 +1,36 @@
+'use strict';
+
+let co  = require('co');
+let cli = require('heroku-cli-util');
+
+function formatKey (key) {
+  key = key.trim().split(/\s/);
+  return `${key[0]} ${key[1].substr(0,10)}...${key[1].substr(-10,10)} ${cli.color.green(key[2])}`;
+}
+
+function* run(context, heroku) {
+  let keys = yield heroku.get('/account/keys');
+  if (context.flags.json) {
+    cli.styledJSON(keys);
+  } else if (keys.length === 0) {
+    cli.warn('You have no ssh keys.');
+  } else {
+    cli.styledHeader(`${cli.color.cyan(keys[0].email)} keys`);
```

Comment:
@dickeyxxx The authenticated user email should be printed here, as the key may have a different email associated with it.